### PR TITLE
exim: pcap/pacpng import Start + Menu Item

### PR DESCRIPTION
**Pull request by Sebitosh**
## Overview
Start of the work on the pcap import project. Added a Menu Item, and basic files to start implementing.

## Related Issues
zaproxy/zaproxy#4812

## Checklist
- [ ] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [NA] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [ ] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

**Comment by Sebitosh** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Sebitosh** [comment]
recheck

File: addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties [hunk 1]
```diff
@@ -44,11 +44,17 @@ exim.importLogFiles.choosefile.title = Select Input Type
 exim.importLogFiles.import.menu.label = Import URLs from Logs or raw Files...
 exim.importLogFiles.log.type.modsec2 = ModSecurity2 Logs
 exim.importLogFiles.log.type.zap = ZAP Messages
+exim.importPcap/Pcapng.choosefile.filter.pcap.description = Pcap file (*.pcap)
+exim.importPcap/Pcapng.choosefile.filter.pcapng.description = Pcap file (*.pcapng)
+
+exim.importPcap/Pcapng.topmenu.import = Import a PCAP or PCAPNG file
+exim.importPcap/Pcapng.topmenu.import.tooltip = Import a PCAP or PCAPNG file and add the HTTP(S) traffic inside to the sites tree and history panel.
```
Comment (kingthorin) [inline addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties:51]: I'd suggest dropping the slash in the key name. Just let it be importpcap or import.pcap

File: addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties [hunk 2]
```diff
@@ -44,11 +44,17 @@ exim.importLogFiles.choosefile.title = Select Input Type
 exim.importLogFiles.import.menu.label = Import URLs from Logs or raw Files...
 exim.importLogFiles.log.type.modsec2 = ModSecurity2 Logs
 exim.importLogFiles.log.type.zap = ZAP Messages
+exim.importPcap/Pcapng.choosefile.filter.pcap.description = Pcap file (*.pcap)
+exim.importPcap/Pcapng.choosefile.filter.pcapng.description = Pcap file (*.pcapng)
```
Comment (kingthorin) [inline addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties:48]: I'd suggest using:
PCAP File (*.pcap)
PCAPNG File (*.pcapng)

File: addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties [hunk 3]
```diff
@@ -44,11 +44,17 @@ exim.importLogFiles.choosefile.title = Select Input Type
...
+
+exim.importPcap/Pcapng.topmenu.import = Import a PCAP or PCAPNG file
+exim.importPcap/Pcapng.topmenu.import.tooltip = Import a PCAP or PCAPNG file and add the HTTP(S) traffic inside to the sites tree and history panel.
 
 exim.importurls.topmenu.import = Import a File Containing URLs
 exim.importurls.topmenu.import.tooltip = The file must be plain text with one URL per line.\nBlank lines and lines starting with a # are ignored.
 exim.importurls.warn.scheme = "{0}" does not have a scheme.
 
+
```
Comment (kingthorin) [inline addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties:57]: Revert this blank addition.

File: addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties (hunk 1 above)
Comment (kingthorin) [inline addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties:51]: For thought/discussion do we really need to say PCAP or PCAPNG I know they're technically different but does the greater "industry" treat them as one in the same so to speak?

File: addOns/exim/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- Pcap/ng import GUI (Issue 4812, WIP)
```
Comment (kingthorin) [inline addOns/exim/CHANGELOG.md:8]: I'd suggest just making this generic to cover all the initial work.
`- Initial PCAP import support (Issue 4812).`
or something like that.

File: addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties (hunk 1 above)
Comment (Sebitosh) [reply addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties:51]: Great question, I personally have no clue if people will deduce that if only "Import a PCAP file" is shown. But I agree it's not great to have such a long title. I thought maybe using "Import a PCAP(NG) file" could help but not sure it is an elegant solution either

**Review by kingthorin — APPROVED** [review]
This seems fine to me. I'd remove the WIP from the changelog note, but that's nit picky.

**Comment by kingthorin** [comment]
Needs a spotlessApply.
`./gradlew :aO:exim:spotlessApply`

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**